Minesweeper.cpp

```
@@ -1,16 +1,137 @@
+		}
+	}
+
+	bool ShouldRenderSquare(IntVector2 coords) override
+	{
+		return !m_grid[coords.y][coords.x].isCovered;
+	}
+
+public:
+	Minesweeper(IntVector2 dimensions, GameBoard::Square sampleTile) : GameBoard(dimensions, sampleTile)
+	{
+		m_grid = GenerateBoard<TileGrid, Tile>(IntVector2{ 9,9 }, sampleTile);
```

cppcheck (2.10-2)

[performance] Variable 'm_grid' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)